backend/dace_vscode/transformations.py

```diff
@@ -42,7 +42,7 @@ def expand_library_node(json_in):
         if cfg_id is None:
             sdfg.expand_library_nodes()
         else:
-            if DACE_VERSION >= '0.16.0':
+            if DACE_VERSION >= '0.16.0' or not hasattr(sdfg, 'sdfg_list'):
```

Comment:
These are unsafe checks, ask instead if hasattr cfg_list. Also to be on the safe side with string comparisons you can make DACE_VERSION an int tuple once and check against (0, 16, 0)

Reply:
Why is the string comparison unsafe? Typescript / Javascript compares strings with regular numerical ordering, which means semver version strings can be compared as-is.

Reply:
With the cfg_list check the version can be dropped entirely.